includes/admin/reports/reports.php

```diff
@@ -49,7 +49,7 @@ function give_reports_page() {
 					'tab'              => 'export',
 					'settings-updated' => false,
 				), $current_page ) ); ?>" class="nav-tab <?php echo 'export' === $active_tab ? esc_attr( 'nav-tab-active' ) : ''; ?>"><?php esc_html_e( 'Export', 'give' ); ?></a>
-			<?php }
+			<?php } ?>
```

Comment:
I don't think the php tag should be closed here. On line 60 the `do_action` function is called and there are no other php opening tags.

Reply:
You're right - forgot to cleanup after myself with this reports.php file. I'll make sure to keep a closer eye on things before I pass them up for QA!